src/platform.js

```diff
@@ -44,16 +44,40 @@ export class Platform {
    * @return {boolean}
    */
   isSafari() {
-    return /Safari/i.test(this.win.navigator.userAgent) && !this.isChrome();
+    return /Safari/i.test(this.navigator.userAgent) && !this.isChrome();
   }
 
   /**
-   * Whether the current browser a Chrome browser.
+   * Whether the current browser is a Chrome browser.
    * @return {boolean}
    */
   isChrome() {
     // Also true for MS Edge :)
-    return /Chrome|CriOS/i.test(this.win.navigator.userAgent);
+    return /Chrome|CriOS/i.test(this.navigator.userAgent);
+  }
+
+  /**
+   * Whether the current browser is a Facebook webview.
+   * @return {boolean}
+   */
+  isFacebook() {
```

Comment:
But it's the intent, right? In other words: "the document is viewed under FB or Twitter, etc". It's possible that we will eventually add more meaning to viewer, but otherwise it feels to dangerous to just keep adding overlapping classes such as `amp-referer-`, `amp-service-`, `amp-viewer-` that would otherwise mean very similar things.